### DISCUSSION TITLE
iminuit insert many initial parameters

### DISCUSSION THREAD

**Discussion by EkVlachoudi** [scikit-hep/iminuit#712] [Q&A]
Hello,
i'd like to ask what's the method to insert initial parameters in iminuit , not just a value for each parameter, but a list of values for each parameter. I have 3 parameters and 10 values for each parameter.When it comes to pyroot i did it with the command mnparm(m.mnparm(i, "name", value, error, low limit, up limit, ierflg) ) for all the different values for each parameter. Is there a similar way to do this? 
I dont know if it's the riht place to ask this but no one answered in other forums.
Thank you!
Irene

**Comment by HDembinski** [comment]
In iminuit, a parameter can have only one value. You can set several parameters at once you assigning an array to Minuit.values, this is explained in the basic tutorial.
